src/Jackett.Common/Indexers/Definitions/AnilibriaTop.cs

```diff
@@ -0,0 +1,139 @@
+        public override string SiteLink { get; protected set; } = "https://anilibria.top/";
+        private string ApiBase => $"{SiteLink}api/v1/";
+        public override string Language => "ru-RU";
+        public override string Type => "public";
+        public override TorznabCapabilities TorznabCaps => SetCapabilities();
+
+        public AnilibriaTop(IIndexerConfigurationService configService, WebClient wc, Logger l, IProtectionService ps,
+                            ICacheService cs) : base(
+            configService: configService, client: wc, logger: l, p: ps, cacheService: cs,
+            configData: new ConfigurationDataAnilibriaTop())
+        {
+        }
+
+        private static TorznabCapabilities SetCapabilities()
+        {
+            var caps = new TorznabCapabilities
+            {
+                TvSearchParams = new List<TvSearchParam> { TvSearchParam.Q, TvSearchParam.Season, TvSearchParam.Ep }
+            };
+            caps.Categories.AddCategoryMapping("TV", TorznabCatType.TVAnime, "Аниме TV");
+            caps.Categories.AddCategoryMapping("MOVIE", TorznabCatType.TVAnime, "Аниме Фильмы");
+            caps.Categories.AddCategoryMapping("OVA", TorznabCatType.TVAnime, "Аниме OVA");
+            caps.Categories.AddCategoryMapping("ONA", TorznabCatType.TVAnime, "Аниме ONA");
+            caps.Categories.AddCategoryMapping("SPECIAL", TorznabCatType.TVAnime, "Аниме Спешл");
+            caps.Categories.AddCategoryMapping("WEB", TorznabCatType.TVAnime, "Аниме WEB");
+            caps.Categories.AddCategoryMapping("OAD", TorznabCatType.TVAnime, "Аниме OAD");
+            caps.Categories.AddCategoryMapping("DORAMA", TorznabCatType.TV, "Дорамы");
+            return caps;
+        }
+
+        public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken configJson)
+        {
+            LoadValuesFromJson(configJson);
+            IsConfigured = false;
+
+            try
+            {
+                var results = await PerformQuery(new TorznabQuery());
+
+                if (!results.Any())
+                {
+                    throw new Exception("API unavailable or unknown error");
+                }
+
+                IsConfigured = true;
+                SaveConfig();
+            }
+            catch (Exception e)
+            {
+                throw new ExceptionWithConfigData(e.Message, configData);
+            }
+
+            return IndexerConfigurationStatus.Completed;
+        }
+
+        protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
+        {
+            var releases = new List<ReleaseInfo>();
+            var template = Uri.EscapeDataString(query.GetQueryString());
+
+            if (query.IsTest)
+            {
+                template = "*";
+            }
+            else if (string.IsNullOrEmpty(template))
+            {
+                return releases;
+            }
```

Comment:
I think the fix for the config save error I am getting might be:
```suggestion
            if (string.IsNullOrEmpty(template))
            {
                template = "*";
            }
```

Also, In response to a TEST or a keywordless search by either a user's manual search or an *arr app doing an indexer check, the indexer should return a short list of the latest releases.
If that is not possible, we often use the current year as a search query, or use some other default keyword that will return one or more titles.
I presume a query using an asterisk on this API returns something? 
Because no results is considered a failure for a TEST.